nixos/modules/module-list.nix

```
@@ -210,7 +210,7 @@
   ./programs/extra-container.nix
   ./programs/fcast-receiver.nix
   ./programs/feedbackd.nix
-  ./programs/firefox.nix
+  ./programs/firefox
```

```suggestion
  ./programs/firefox/default.nix
```
AFAIK `default.nix` is usually specified explicitly even though it isn't technically needed